Dockerfile.test

```diff
@@ -24,6 +24,8 @@ COPY auth/nodemon.json ./
 COPY auth/tsconfig*.json ./
 COPY auth/wait-for.sh ./
 
+RUN chmod -R 777 .
```

Comment:
This is copied from the backend to prevent docker changing the ownership of files during linting etc: https://github.com/OpenSlides/openslides-backend/blob/9e6879c6325455f4e156180f69f2433902a17b70/dev/Dockerfile.dev#L23C1-L23C1